src/main/java/org/folio/circulation/domain/representations/StoredFeeFineAction.java

```diff
@@ -20,7 +21,7 @@ public StoredFeeFineAction(StoredFeeFineActionBuilder builder) {
     this.put("userId", builder.userId);
     this.put("accountId", builder.accountId);
     this.put("source", builder.createdBy);
-    this.put("createdAt", builder.createdAt);
+    write(this, "createdAt", builder.createdAt);
```

Comment:
If we are going to use write for some of this, I think it would be good to use `write` for all of the writing to JSON in this method.

Reply:
Changed now.